### ISSUE TITLE
Contradiction between book's suggestions related to python imports and flake8 which the book recommends

### ISSUE THREAD

**Issue by ramast** [feldroy/two-scoops-of-django-3.x#35]
According to wemake-python-styleguid you shouldn't use local imports "for consistency". fluke8 raises [LocalFolderImportViolation](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.LocalFolderImportViolation) if you do so.

Not a big deal but  I think it's worth a mention in the book.

**Comment by ramast** [comment]
Also while in the same topic, some of the recommendations by the library contradict with Django's styling for example it wants `class Meta(object):` instead of `class Meta`. and it gives error when using attribute `objects` because it's blacklisted

```
17:50 backend $ flake8 orders/models.py

orders/models.py

  95:5     WPS110 Found wrong variable name: objects
  objects = OrderQuerySet.as_manager()
```

I've spent some time googling how to whitelist this word since I don't want to fully disable this rule but no luck so far.

I think if the book is going to recommend wemake-python-stylegui then please give more information on how to make it usable in a django project.

**Comment by pydanny** [comment]
Yeah, I'm not the biggest fan of wemake-python-styleguide either. We're taking it out as of today's release.

Sorry for putting you through that.  If we ever meet, I owe you ice cream, or a drink.

**Comment by pydanny** [comment]
What's your full name so we can give you proper attribution in the book?

**Comment by pydanny** [comment]
Nevermind, found you on Linkedin. 😉

**Comment by ramast** [comment]
Sweet, Thank you!

I've made some research about this and found that these two lines

     extend-ignore = WPS432,WPS305,WPS300,WPS601,WPS317,WPS318,WPS319,WPS306,WPS529,WPS602,E800,D,
    allowed-domain-names = objects

solved most of problems (plus others).

Regarding attribution I'd be much happier if you have a look at an alternative I've wrote to Django Choices, an app that you guys have suggested in your previous (django 1.5 book)

https://github.com/ramast/django_model_helpers

If you found it useful, It would mean the world to me if you include it. Otherwise the honourable mention is still awesome!
